python/cudnn/gemm/cutedsl/grouped/unfused/api.py

```
@@ -142,6 +153,22 @@ def _tensor_signature(tensor: Optional[torch.Tensor], *, dynamic_m: bool = False
     )
 
 
+def _fastsig(tensor: Optional[torch.Tensor], *, dynamic_m: bool = False) -> Optional[tuple]:
```

Ruff (0.16.1)

[error] Undefined name `torch`

(F821)